core/src/java/main/org/jaxen/expr/DefaultFunctionCallExpr.java

```diff
@@ -77,7 +77,7 @@ public void addParameter(Expr parameter)
     }
 
 
-    public List getParameters()
+    public List<Expr> getParameters()
```

Comment:
This is a potentially code breaking public API change, which we won't do. (Consider subclasses that override this method.) Ditto for other signature changes below.

Reply:
I have done a little test.
I have written an class OverwriteTest like so:
```
package org.jaxen.expr;

import java.util.List;

public class OverwriteTest extends DefaultFunctionCallExpr {
    public OverwriteTest(String prefix, String functionName) {
        super(prefix, functionName);
    }

    @Override
    public List getParameters() {
        return super.getParameters();
    }
}
```
No Problem for the Java Compiler indeed it is not a good practice to drop the generics but it is possible without an API break.

If you agree the signature in the interface `FunctionCallExpr` should also be changed.